js/views/modals/editListing/EditListing.js

```diff
@@ -1166,6 +1166,8 @@ export default class extends BaseModal {
             tags: item.max.tags,
             photos: this.MAX_PHOTOS,
           },
+          peerID: app.profile.id,
```

Comment:
This was already part of the listing data passed in. There's a nested vendorID object which has a peerID.